features/minimal_crash.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Minimal Crash report
+
+Scenario: Crash within the crash handler
+    When I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+    And I configure the app to run on "iPhone 8"
+    And I crash the app using "MinimalCrashReportScenario"
+    And I relaunch the app
+    Then I should receive a request
+    And the request is a valid for the error reporting API
+    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
+    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
+    And the payload field "events" is an array with 1 element
+    And the exception "errorClass" equals "Exception"
+    And the exception "type" equals "cocoa"
+
+    And the event "breadcrumbs" is null
+    And the event "metaData.error" is null
+    And the event "context" is null
+    And the event "user.id" is null
+    And the event "context" is null
+    And the event "severity" equals "error"
+    And the event "severityReason.type" equals "unhandledException"
+
+    And the event "app.bundleVersion" equals "1"
+    And the event "app.version" equals "1.0"
+    And the event "app.type" equals "iOS"
+    And the event "app.id" equals "com.bugsnag.iOSTestApp"
+    And the event "app.releaseStage" equals "development"
+
+    And the event "device.locale" is not null
+    And the event "device.osVersion" is not null
+    And the event "device.model" is not null
+    And the event "device.jailbroken" is not null
+    And the event "device.modelNumber" is not null
+    And the event "device.osName" is not null
```

Comment:
The OS name should be a constant value here, so we can use "equals" rather than "is not null"